src/BizHawk.Client.Common/Api/Classes/SQLiteApi.cs

```diff
@@ -1,42 +1,45 @@
-﻿using System.Collections.Generic;
-using System.Data;
-using System.Data.SQLite;
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+using Microsoft.Data.Sqlite;
 
 namespace BizHawk.Client.Common
 {
 	public sealed class SQLiteApi : ISQLiteApi
 	{
-		private SQLiteConnection _dbConnection;
+		private SqliteConnection _dbConnection;
 
 		public string CreateDatabase(string name)
 		{
 			try
 			{
-				SQLiteConnection.CreateFile(name);
+				File.Create(name).Dispose();
```

Comment:
If the equivalent of this in the new library is a no-op, there's no point checking it. We assume paths are good and let them throw on first use.

Reply:
This is the equivalent of what the old library did (well, old library did `File.Create(name).Close();` technically, but Dispose() is probably more proper)